outgoingra/src/main/java/no/bankid/outgoing/ra/RaRequirements.java

```diff
@@ -6,139 +6,319 @@
-                    example = "vipps-bapp-client",
-                    required = true)
-            @HeaderParam("X-CLIENT-CLIENTNAME") String clientName,
-            @Parameter(description = "Originator/bank-number for endUser's bankID, valid BankID ODS number",
-                    example = "3625",
-                    required = true)
-            @HeaderParam("X-DATAOWNERORGID") String odsBankNo,
-            @Parameter(description = "EndUser's Norwegian national identification number, string of 11 digits",
-                    example = "11111111016",
-                    required = true)
-            @HeaderParam("X-CUSTOMERID") String nnin
+            @Parameter(description = DESCRIPTION_SIGNATURE,
+                    example = EXAMPLE_SIGNATURE,
+                    required = true)
+            @HeaderParam(SIGNATURE) String httpSignature,
+            @Parameter(description = DESCRIPTION_DATE,
+                    example = EXAMPLE_DATE,
+                    required = true)
+            @HeaderParam(DATE) String date,
+            @Parameter(description = DESCRIPTION_DIGEST,
+                    example = EXAMPLE_DIGEST,
+                    required = true)
+            @HeaderParam(DIGEST) String digest,
+            AuthenticationBody authenticationBody
+    );
+
+    /////////////////////////////// TODO: reserven�kkel
+
+    @Operation(
+            summary = "Health check of the RA application",
+            description = "Checks that the RA is capable of handling endpoints declared here",
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "RA is healthy"),
+                    @ApiResponse(
+                            responseCode = "500",
+                            description = "RA is not healthy")}
+            , tags = {"Activation without Code Device"})
+    @Path("healthcheck")
+    @Produces(MediaType.APPLICATION_JSON)
+    @GET
+    Response healthCheck();
+
+    @Operation(summary = "Check two-channel options for endUser"
+            , description = "<p>Endpoint to check if a specific user is eligible for two-channel activation. A user is eligible if:</p><ul><li>They have an active BankID with the current RA.</li><li>They have at least two independent verified points of contact, not violating the following disallowed combinations.<ul><li>sms + phone call</li><li>digipost + post</li><li>the same method used twice, even with different hints.</li></ul></li><li>They have not <span style=\"color: rgb(32,31,30);\">reissued the BankID (password reset)</span> in the last X days. (Subject to regulation by Bits).</li></ul><p>Which method types are supported is up to the individual RA to decide, but at least two independent (i.e. not terminating in the same point) types must be supported. i.e. sms + email is ok, phone call and sms to the same number is not. </p>"
```

Comment:
```suggestion
            , description = "<p>Endpoint to check if a specific user eligibility from single originator for self-service activation.</p><p>The RA should check if the provided phone number is registered for the user, but return the other information regardless."
```

Reply:
Mener du ikke: 

Endpoint to check if a specific endUser is eligible for self-service activation for the given originator.  

?

Reply:
Jo, jo, det mener jeg.